lib/checks/keyboard/frame-focusable-content-evaluate.js

```diff
@@ -1,7 +1,26 @@
 import isFocusable from '../../commons/dom/is-focusable';
 
+export default function frameFocusableContentEvaluate(
+  node,
+  options,
+  virtualNode
+) {
+  if (!virtualNode.children) {
+    return undefined;
+  }
+
+  try {
+    return !virtualNode.children.some(child => {
+      return focusableDescendants(child);
+    });
+  } catch (e) {
+    return undefined;
+  }
+}
+
 function focusableDescendants(vNode) {
-  if (isFocusable(vNode)) {
+  const tabIndex = parseInt(vNode.attr('tabindex'), 10);
+  if (isFocusable(vNode) && (isNaN(tabIndex) || tabIndex > -1)) {
```

Comment:
Probably put these in reverse order, for performance:

```suggestion
  if ((isNaN(tabIndex) || tabIndex > -1) && isFocusable(vNode)) {
```